hydroshare.re

```diff
@@ -30,7 +30,11 @@ acPostProcForCopy () {
 
 # catch "iput" command
 acPostProcForPut () {
-    msiHSAddNewFile($objPath, "/hydroshareZone/home/cuahsiDataProxy/bags", "quotaUserName", "Federated", "hsuser", "dummy", "local.hs.org/hsapi/_internal/update_quota_usage/"); 
+    # work around to not call msiHSAddNewFile in the case of irepl and only call it in the case of iput
```

Comment:
Please add another comment for posterity:

`# $oprType == 1 for iput; $oprType == 6 for irepl`

Reply:
Good idea - have added this info in the comment. See if it looks good now. @cbcunc